doc/source/theory/N3LO_ad.rst

```diff
@@ -326,7 +326,7 @@ The following tables list all the considered input functions.
         *   - :math:`f_2(N)`
             - :math:`\frac{S_1^3(N)}{N}`
         *   - :math:`f_3(N)`
-            - :math:`\frac{1}{N-1},\ \mathcal{M}[\ln^3(1-x)]`
+            - :math:`\frac{1}{N^4},\ \mathcal{M}[\ln^3(1-x)]`
```

Comment:
I assume this is the major change in this commit, correct @giacomomagni ? (i.e. all else is just the needed propagation) with that you broke your rule 3 from above - is there a physics/math reason? or the uncertainty is just more well behaved?

Reply:
yes this is correct. For `P_qg` there are at least 3 different possibilities to select `f_3` which are: 
`1/N^4, 1/(N-1)` , `1/N^4, lm13(N)` , `1/(N-1),  lm13(N)` and so on.. in each case you can then choose to drop or not 1/2 outliers (see above).From what I've tried uncertainties can change from [this more conservative](https://github.com/NNPDF/eko/files/10260288/gamma_gq_msht_logx.pdf)
 (what we have now) to something more "aggressive" [like this](https://github.com/NNPDF/eko/files/10260291/gamma_qg_msht_logx_old.pdf). 
 Since I have no reasons to say that error bands should be that smaller, and I thought it was wiser to have larger uncertainties for the moment, such that the scaling is included and there is a smaller tension with MSHT. 
I'm open to change idea. 

Regarding rule 3, I'd say it has to be updated since if you interpret the rule this way it's broken also for `P_gq`... 
I was just too lazy to say that we draw all the possible candidates and then we cherry pick the most representative one. I'll improve the docs :)